common/views/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -25,7 +25,7 @@ const Breadcrumb: FunctionComponent<BreadcrumbItems> = ({ items }) => {
   ];
 
   return (
-    <BreadcrumbWrapper>
+    <BreadcrumbWrapper className="is-hidden-print">
```

Comment:
Is this the breadcrumb at the top of the page?

I'm not sure this should be hidden; it's a useful navigation element for finding your way back to something later.

Reply:
It is! And not a bad shout, I'd happily re-add it if we think they could serve that purpose.

Reply:
I've not followed the breadcrumb discussion in a bit and I know more has happened; do we believe they currently could serve that purpose/are useful in that sense @jennpb ?

Reply:
We've not done anything to improve the breadcrumbs yet @rcantin-w. I'm not certain they are useful in a physical printout, especially given that our breadcrumbs aren't path-based. A user couldn't use the breadcrumbs as a reference to know how they got to that page. In that instance, either the page title or the URL is a more meaningful. I would see value in hiding it.

Reply:
FWIW I feel like these should definitely be hidden in print